### ISSUE TITLE
Add exceptions

### ISSUE THREAD

**Issue by favonia** [RedPRL/sml-redprl#619]

**Comment by jonsterling** [comment]
Do you mean, to the computation system?

**Comment by favonia** [comment]
Yes. We probably have to wait for @ecavallo's Immortal account of partial functions to fully utilize them, but we can prepare ourselves before that.

**Comment by jonsterling** [comment]
I love it! We can use it to realize continuity principles... and maybe Markov’s principle?
